examples/ui/scaling.rs

```diff
@@ -0,0 +1,144 @@
+//! This example illustrates the UIScale resource from bevy_ui
```

Comment:
I think it's `UiScale`, it would also be nice if this would link to the element (or does this not work in examples?)

Reply:
Not really sure. I'll make the change regardless